Change the call + FAQs (#595)
ropensci/roweb3: content/blog/2023-07-14-call-2023-champions-program/index.md

---
title: Open Call for rOpenSci Champions Program Applications! 
author: Yanina Bellini Saibene
date: '2023-07-14T00:00:01-07:00'
slug: call-champions-program
tags:
  - community
  - help
  - beginner
  - welcome
  - champions program
featured: true 

> description: We open the call for the second cohort of Champions and Mentors

Reviewer: ```suggestion
description: We open the call for the second cohort of Champions and Mentors!
```

## What is our Champions Program?

We have developed a **12-month-long program including cohort-based training, project development, and personal mentorship**. 

Champions will participate in a 6-week long cohort-based training on a) planning and facilitating engaging and inclusive workshops to support participants’ success, b) knowledge of various channels through which new members can engage in and contribute to rOpenSci and R projects, and c) technical skills in software development and review. 

After training, Champions will develop their project, which can be one of these options: 

- create a new package, 
- go through the review process with an R package they already have developed, 
- or be a reviewer for our peer review system.  

Other Champions activities include options for delivering a talk, writing blog posts and teaching other community members. During this stage, Champions will receive personal and group mentorship through meetings with mentors, quarterly cohort calls, Community Manager support, and a dedicated channel on rOpenSci Slack workspace.

We will provide a $1000 (USD) stipend to the selected community champions.

> We are looking for **six to nine** Champions for this second cohort. We expect to co-create the rOpenSci Champions Program with the selected applicants and influence future editions with their experience and feedback.

Reviewer: ```suggestion
We are looking for **six to nine** Champions for this second cohort. We expect to co-create this iteration of the rOpenSci Champions Program with the selected applicants and to refine future editions with their experience and feedback.
```

We look forward to receiving many applications and are committed to directing non-selected applicants towards other ways in which they can contribute to rOpenSci.

> You can see all details on our [Champions Program webpage](/champions/).

Reviewer: ```suggestion
```

## How to apply?

> Please, fill out the [application form](https://airtable.com/shrAsYlSXU0coJ5Ld) (this [template](/champions/files/champions_template) shows the information we require) before __Friday, Nov 7, 2022__.
> 
> The application forms have to be filled in _English_.
> 
> ### Application timeline
> 
> |Date|Event|
> |----|-----|
> |Sep 23, 2022|Call for Applications opens|
> |Oct 24, 2022| [OPTIONAL] [Community Call](/commcalls/oct2022-champions/) -  Tips for submitting your Champions Program application |
> |Nov 1, 2022| [OPTIONAL] [Co-working hours](/events/coworking-2022-11/): receive help with your application to the Champions Program |
> |Nov 7, 2022|Call for applications closed|
> |Nov 25, 2022|Communication with selected Champions |
> 

Reviewer: I'd consider directing people to the Champions page to avoid having too many different versions of the details scattered around. 

```suggestion
See the [Champions Program page](/champions/) for all details regarding applying, timelines and eligibility. 
```

Author: I see a good point on what you said, but this is the blog post with the announcement, I like to have the info there and avoid another more click for people to go and look for the info.

## Do you have any questions?

Please, contact our Community Manager [by email](mailto:yabellini@ropensci.org). 

> We will have a [Community Call on October 24](/commcalls/oct2022-champions/) and a [Coworking space on November 1](/events/coworking-2022-11/) to help you with your application.

Reviewer: Update this dates/links

> We will have a [Community Call on October 24](/commcalls/oct2022-champions/) and a [Coworking space on November 1](/events/coworking-2022-11/) to help you with your application.

Author: ```suggestion
We will have a [Community Call on Tuesday, 25 July 2023 14:00 UTC](/commcalls/july2023-championprogram/) and a [Coworking space on august](/events/coworking-2023-08/) to help you with your application.
```
